cmd/kflex/ctx/ctx.go

```diff
@@ -79,8 +80,14 @@ func (c *CPCtx) Context(chattyStatus, failIfNone bool) {
 				fmt.Fprintf(os.Stderr, "Error loading kubeconfig context from server: %s\n", err)
 				os.Exit(1)
 			}
-			if err = kubeconfig.SwitchContext(kconf, c.Name); err != nil {
-				fmt.Fprintf(os.Stderr, "Error switching kubeconfig context after loading from server: %s\n", err)
+			// context exists only for CPs that are not of type host
+			if c.Type != tenancyv1alpha1.ControlPlaneTypeHost {
+				if err = kubeconfig.SwitchContext(kconf, c.Name); err != nil {
+					fmt.Fprintf(os.Stderr, "Error switching kubeconfig context after loading from server: %s\n", err)
+					os.Exit(1)
+				}
+			} else {
+				fmt.Fprintf(os.Stderr, "control plane %s is of type 'host', cannot switch context\n", c.Name)
```

Comment:
I would not characterize this as a failure; the context has been successfully switched to the context of the requested control plane.